### DISCUSSION TITLE
I unstar this topic long time ago.

### DISCUSSION THREAD

**Discussion by gaelthas** [community/community#38649] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/11371061/200760897-fe54a2ba-a6d3-4975-972b-119b5e3937aa.png)
Why do I keep getting recommendations like this?

**Comment by ossanna16** [comment]
Hi there @gaelthas and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
